scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/ActivityInstance.java

```diff
@@ -163,9 +163,27 @@ public Duration getDuration() {
   }
 
   public Duration getEndTime() {
+    if(!hasDuration()){
```

Comment:
:large_blue_circle: My preference would be that such changes are contained in a separate commit, given that I think this goes further than linting/cleaning. I'm happy and open to be convinced otherwise 😄

Reply:
Thanks for the note. It helps my understanding. 👍